src/App.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
+import { Admin, Resource } from "react-admin";
+import simpleRestProvider from "ra-data-simple-rest";
+import { UserList, UserEdit } from "./users";
 
-function App() {
+const App = () => {
   return (
-    <div>
-      <h1>Helo</h1>
-    </div>
+    <Admin dataProvider={simpleRestProvider("http://localhost:8000/admin")}>
```

Comment:
change this localhost to an environment variable of our api, ask your team if you need help with this
## DO NOT PUSH THE CONFIG.ENV FILE